operate/quickstart/run_service.py

```diff
@@ -113,16 +126,17 @@ def load_local_config() -> QuickstartConfig:
 def configure_local_config(template: ServiceTemplate) -> QuickstartConfig:
     """Configure local quickstart configuration."""
     config = load_local_config()
-
+    attended = os.getenv("ATTENDED", "true").lower() == "true"
```

Comment:
this is not needed anymore

Reply:
after removing this variable, the rest of the code can be made same as before, with the only difference being `ask_or_get_from_env` in place of `input` and `getpass`